apps/sms/views/shared/style/root.css

```diff
@@ -12,7 +12,7 @@ body {
   font-size: 1.6rem;
   line-height: 2.2rem;
   vertical-align: middle;
-  background-color: #FFF;
+  background: var(--header-background);
```

Comment:
> That said, I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1228379 :) Not sure it would be fixed anytime soon...

Thanks! Hopefully it will be fixed, otherwise it's really confusing that header color is considered as the "main" color :) Not all apps/views even have a header :)